app/models/peep.rb

```diff
@@ -0,0 +1,10 @@
+class Peep
+
+  include DataMapper::Resource
+
+  has n, :makers, :through => Resource
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)